src/main/java/ca/on/oicr/gsi/visionmate/ServerConfig.java

```diff
@@ -15,7 +15,10 @@
  */
 public class ServerConfig {
   
-  public static enum SortOrder {ROWS, COLUMNS};
+  public enum SortOrder {ROWS, COLUMNS};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)